lib/java/com/google/android/material/textfield/TextInputLayout.java

```diff
@@ -187,6 +187,8 @@
  */
 public class TextInputLayout extends LinearLayout {
 
+  static final int NO_HELPER_TEXT_HORIZONTAL_PADDING = -1;
```

Comment:
The constant name is not accurate. Maybe just `USE_DEFAULT_PADDING`?

Reply:
995e5e7